src/EntityManager.php

```diff
@@ -165,7 +165,7 @@ public function getExpressionBuilder(): Expr
 
     public function beginTransaction(): void
     {
-        $this->conn->beginTransaction();
+        $this->conn->beginTansation();
```

Comment:
this change is totally broken

Reply:
You're reviewing faster than I could revert broken commits. :-D